tox.ini

```diff
@@ -1,18 +1,24 @@
 [tox]
 envlist =
-  py39-django42-wagtail62
-  py{310,311,312}-django{50,51}-wagtail62
-  py312-django42-wagtail{52,60,61,62}
+  py{39,310,311,312,313}-django42-wagtail62
+  py{310,311,312,313}-django{50,51}-wagtail62
+  py{313}-django42-wagtail{52,62}
```

Comment:
non-blocking: best to do Watail LTS with the Django LTS (so `-django42-wagtail{52,62}`. This also works, just that neither Django 4.2, nor Wagtail 5.2 declare official support for Python 3.13

Reply:
Hmm, I wonder how py3.13 got in there. I've messed with the matrix too much and lost track of things. So I've added a couple of comments to explain what each section should do. Thanks for flagging this!